src/test/java/com/wikia/webdriver/testcases/searchtests/CrossWikiSearchTests.java

```diff
@@ -44,21 +45,24 @@ public void crossWikiSearch_002_pagination() {
     // verify results pos parameter for first page
     search.verifyResultsPosForPage(0, SearchContent.RESULTS_PER_PAGE);
     search.verifyResultsCount(SearchContent.RESULTS_PER_PAGE);
-    search.verifyThumbnails(SearchContent.RESULTS_PER_PAGE);
+    search.verifyThumbnailsAmount(SearchContent.RESULTS_PER_PAGE);
+    Assertion.assertTrue(search.areThumbnailsContainImages(), "Thumbnail does not contain image");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Thumbnail does not contain image" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)